hermes-console/static/partials/constraintsList.html

```diff
@@ -0,0 +1,59 @@
+<section ng-controller="ConstraintsListController">
+
+    <h1><small>Constraints:</small></h1>
+
+    <div class="row">
+        <div class="list-group">
+            <div class="list-group-item">
+                <div class="pull-right">
+                    <button class="btn btn-primary">
+                        <span class="glyphicon glyphicon-plus"></span>
+                    </button>
+                </div>
+                <h4>
+                    Topic constraints
+                </h4>
+            </div>
+            <a ng-href="#/constraints/{{item.topicName}}" class="list-group-item" ng-repeat="item in topicConstraints">
+                <small>{{$index + 1}}</small>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;
+                <strong>{{item.topicName}}</strong>
+                <div class="pull-right">
+                    <span class="label label-info">consumers number: {{ item.consumersNumber }}</span>
+                    <span class="glyphicon glyphicon-chevron-right"></span>
+                </div>
+            </a>
+        </div>
+
+        <div ng-show="fetching" class="text-center">
+            <h1><i class="fa fa-circle-o-notch fa-spin"></i></h1>
+        </div>
+    </div>
+
+    <div class="row">
+        <div class="list-group">
+            <div class="list-group-item">
+                <div class="pull-right">
+                    <button class="btn btn-primary">
+                        <span class="glyphicon glyphicon-plus"></span>
+                    </button>
+                </div>
+                <h4>
+                    Subscription constraints
+                </h4>
+            </div>
+            <a ng-href="#/constraints/{{item.subscriptionName}}" class="list-group-item" ng-repeat="item in subscriptionConstraints">
+                <small>{{$index + 1}}</small>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;
+                <strong>{{item.subscriptionName}}</strong>
+                <div class="pull-right">
+                    <span class="label label-info">consumers number: {{ item.consumersNumber }}</span>
+                    <span class="glyphicon glyphicon-chevron-right"></span>
+                </div>
+            </a>
+        </div>
+
+        <div ng-show="fetching" class="text-center">
```

Comment:
does it ever show?

Reply:
I added this div by accident when I copied code. I will remove it.

Reply:
I removed it.